assets/jsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I'm not sure we want changes made to this. I find when I have problems with local builds, it updates this file and I always exclude it from my commits. Might want to check with @edward-prosser